app/views/assignments/_form.html.slim

```diff
@@ -1,5 +1,6 @@
 = simple_form_for(@assignment) do |f|
   = simple_error_notice f
+  = f.input :user_id, as: :hidden
```

Comment:
Not sure, but this field is ignored because the user is overriden in the controller:

```ruby
@assignment.user = current_user
```

It usually makes sense to do that, otherwise the user can theoretically change the ID in the hidden field and create a record for another user.